packages/main/src/MultiComboBox.js

```diff
@@ -426,17 +426,15 @@ class MultiComboBox extends UI5Element {
 		this.fireEvent("change");
 	}
 
-	_showMorePopover() {
-		this.filterSelected = true;
-		this._toggleRespPopover();
+	togglePopover() {
+		this.allItemsPopover.toggle(this);
 	}
 
-	togglePopover() {
-		if (!isPhone()) {
-			this._inputDom.focus();
-		}
+	_showFilteredItems() {
+		this.filterSelected = true;
+		this._showMorePressed = true;
```

Comment:
When selected items popover is opened and an item is deselected, it is removed from the popover -  is that expected because in UI5 it is just unselected and the token is removed but it remains in the popover. If that is expected, when all the selected items are deselected the popover remains open and empty

Reply:
If the user open the N-more popover and then type something in the input field, just the selected items popover could be filtered. If the typed value does not match an item from the N-more popover, value state error is presented instead of showing the suggestions if there are matching ones. You can compare it with UI5. Something more, for example check the sample "MultiComboBox with items" in the test page. Open the drop down, select all the items starting with "L" and then type "C". Click somewhere else in order to move the focus from the multicombobox and to close the dropdown. Then press over the N-more indicator -  an empty popover is presented, because it tries to filter the selected items with "C" and such items are not presented.

Reply:
Check the first sample in the test page on mobile. Open the N-more popover, deselect one of the suggestions from its checkbox and press the Cancel button of the dialog - the change is not canceled and the selected items are updated